### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by nathanielhu**
Circuit schematic and PCB files

### CODE REVIEW

**Comment by nathanielhu** [comment]
Sometimes I sit and think, and sometimes I just sit

**Comment by aldenhaase** [comment]
Every PR must come with a nice quote from now on

File: KBV3/KBV3-backups/KBV3-2024-03-21_144535.zip
Comment (aldenhaase) [inline KBV3/KBV3-backups/KBV3-2024-03-21_144535.zip:1]: What are all of these giant backup files?

Comment (nathanielhu) [reply KBV3/KBV3-backups/KBV3-2024-03-21_144535.zip:1]: They’re the back up files

**Pull request closed without merging**